extensions/EXT/SPV_EXT_replicated_composites.asciidoc

```diff
@@ -0,0 +1,170 @@
+SPV_EXT_replicated_composites
+=============================
+
+Name Strings
+------------
+
+SPV_EXT_replicated_composites
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Kevin Petit, Arm Ltd.
+- Jeff Bolz, NVIDIA
+- Alan Baker, Google
+- Yuehang Wu, Arm Ltd.
+
+Notice
+------
+
+Copyright (c) 2024 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+
+- Approved by the SPIR-V Working Group: 2024-04-03
+- Approved by the Khronos Board of Promoters: 2024-05-17
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2024-04-03
+| Revision           | 1
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+version 1.6 Revision 3.
+
+This extension requires SPIR-V 1.0.
+
+Overview
+--------
+
+This extension adds instructions to create composite objects whose
+constituents all have the same value without requiring the value to be
+provided for each constituent.
+
+Extension Name
+--------------
+
+To use this extension within a SPIR-V module, the following
+*OpExtension* must be present in the module:
+
+----
+OpExtension "SPV_EXT_replicated_composites"
+----
+
+Modifications to the SPIR-V Specification, Version 1.6
+------------------------------------------------------
+
+Capabilities
+~~~~~~~~~~~~
+
+Modify Section 3.31, "Capability", adding these rows to the Capability table:
+
+--
+[options="header"]
+|====
+2+^| Capability ^| Depends On
+| 6024 | *ReplicatedCompositesEXT* +
+Uses **OpConstantCompositeReplicateEXT**, **OpSpecConstantCompositeReplicateEXT**, or **OpCompositeConstructReplicateEXT** |
+|====
+--
+
+Instructions
+~~~~~~~~~~~~
+
+Add the following new instructions:
```

Comment:
I think we usually indicate which section each new instruction should go it.

Reply:
Yup, now done.